packages/server/src/lib/email.js

```diff
@@ -117,10 +117,14 @@ function sendWelcomeEmail(email, httpOrigin) {
 
 function getGrantDetail(grant, emailNotificationType) {
     const grantDetailTemplate = fileSystem.readFileSync(path.join(__dirname, '../static/email_templates/_grant_detail.html'));
+
+    const shortDescription = grant.description?.length > 380 ? grant.description.substring(0, 380) : grant.description;
+    const description = shortDescription.replace(/<\/?p[^>]*>/g, '');
```

Comment:
@SDBowen Re 
> I am not certain if the above process to remove these tags is necessary. Our seed data contains Grant descriptions enclosed with `<p>` tags. But, when I look at the Grant data in production, I do not see Grant descriptions with these `<p>` tags. So, unclear if these `<p>` tag enclosed descriptions still exist.

I don't believe that the grant description text should ever include HTML; my suspicion is that the fixtures were based on grant data that was scraped (prior to the `grants-ingest` service), when that may have been more of a possibility. That said, it might be worth keeping, since I don't think there's any guarantee from the grants.gov data source that markup will never be present in a text value.

In fact, it might be worth taking this one step further to remove _any_ HTML (within reason) present in the original `grant.description` text, and then truncate the result to the desired length.

```suggestion
    const description = grant.description.replace(/(<([^>]+)>)/ig, '').substring(0, 380);
```

Reply:
Makes sense and cleaner :+1: